examples/tutorial/metric-learning.ipynb

```diff
@@ -0,0 +1,668 @@
+{
```

Comment:
<p>Would you please add a following citation on the sampling technique introduced here: N-pair sampling (<a href="https://papers.nips.cc/paper/6200-improved-deep-metric-learning-with-multi-class-n-pair-loss-objective" target="_blank">https://papers.nips.cc/paper/6200-improved-deep-metric-learning-with-multi-class-n-pair-loss-objective</a>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/objax/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>